website/docs/r/vapp.html.markdown

```diff
@@ -122,26 +123,58 @@ resource "vcd_vapp" "web" {
 The following arguments are supported:
 
 * `name` - (Required) A unique name for the vApp
-* `catalog_name` - (Optional) The catalog name in which to find the given vApp Template
-* `template_name` - (Optional) The name of the vApp Template to use
-* `memory` - (Optional) The amount of RAM (in MB) to allocate to the vApp
-* `cpus` - (Optional) The number of virtual CPUs to allocate to the vApp
-* `initscript` (Optional) A script to be run only on initial boot
-* `network_name` - (Optional) Name of the network this vApp should join
-* `network_href` - (Optional; **Deprecated**) The vCloud Director generated href of the network this vApp
-  should join. If empty it will use the network name and query vCloud Director to discover
-  this
-* `ip` - (Optional) The IP to assign to this vApp. Must be an IP address or
+* `catalog_name` - (Optional; **Deprecated**) The catalog name in which to find the given vApp Template
+* `template_name` - (Optional; **Deprecated**) The name of the vApp Template to use
+* `memory` - (Optional; **Deprecated**) The amount of RAM (in MB) to allocate to the vApp
+* `cpus` - (Optional; **Deprecated**) The number of virtual CPUs to allocate to the vApp
+* `initscript` (Optional; **Deprecated**) A script to be run only on initial boot
+* `network_name` - (Optional; **Deprecated**) Name of the network this vApp should join
+* `storage_profile` - (Optional)
+* `ip` - (Optional; **Deprecated**) The IP to assign to this vApp. Must be an IP address or
   one of dhcp, allocated or none. If given the address must be within the
   `static_ip_pool` set for the network. If left blank, and the network has
   `dhcp_pool` set with at least one available IP then this will be set with
   DHCP.
 * `metadata` - (Optional) Key value map of metadata to assign to this vApp. Key and value can be any string. (Since *v2.2+* metadata is added directly to vApp instead of first VM in vApp)
-* `ovf` - (Optional) Key value map of ovf parameters to assign to VM product section
+* `ovf` - (Optional; **Deprecated**) Key value map of ovf parameters to assign to VM product section
```

Comment:
May refer to `guest_properties` on vApp or VM so that user is not lost. Something like "use 'guest_properties' block on vApp or VM instead"

Reply:
Well spotted. This was some leftover from copy-and-paste.  Cleaning up